config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -92,6 +92,7 @@ periodics:
     preset-k8s-ssh: "true"
     preset-dind-enabled: "true"
     preset-e2e-kubemark-common: "true"
+    preset-e2e-scalability-containerd: "true"
```

Comment:
containerd doesn't make much sense for kubemark - this will only determine what is running in the base cluster (as we don't run containerd in kubemark cluster). So we don't really care about it.

I would prefer not to touch kubemark wrt containerd.